editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1333,13 +1327,38 @@ SpriteFramesEditor::SpriteFramesEditor() {
 
 	load->connect("pressed", callable_mp(this, &SpriteFramesEditor::_load_pressed));
 	load_sheet->connect("pressed", callable_mp(this, &SpriteFramesEditor::_open_sprite_sheet));
-	_delete->connect("pressed", callable_mp(this, &SpriteFramesEditor::_delete_pressed));
+	delete_frame->connect("pressed", callable_mp(this, &SpriteFramesEditor::_delete_pressed));
 	copy->connect("pressed", callable_mp(this, &SpriteFramesEditor::_copy_pressed));
 	paste->connect("pressed", callable_mp(this, &SpriteFramesEditor::_paste_pressed));
-	empty->connect("pressed", callable_mp(this, &SpriteFramesEditor::_empty_pressed));
-	empty2->connect("pressed", callable_mp(this, &SpriteFramesEditor::_empty2_pressed));
+	empty_before->connect("pressed", callable_mp(this, &SpriteFramesEditor::_empty_pressed));
+	empty_after->connect("pressed", callable_mp(this, &SpriteFramesEditor::_empty2_pressed));
 	move_up->connect("pressed", callable_mp(this, &SpriteFramesEditor::_up_pressed));
 	move_down->connect("pressed", callable_mp(this, &SpriteFramesEditor::_down_pressed));
+
+	load->set_shortcut_context(tree);
+	load->set_shortcut(ED_SHORTCUT("sprite_frames/load_from_file", TTR("Add frame from file"), KeyModifierMask::CMD_OR_CTRL | Key::O));
+	load_sheet->set_shortcut_context(tree);
+	load_sheet->set_shortcut(ED_SHORTCUT("sprite_frames/load_from_sheet", TTR("Add frames from sprite sheet"), KeyModifierMask::CMD_OR_CTRL | KeyModifierMask::SHIFT | Key::O));
+	delete_frame->set_shortcut_context(tree);
+	delete_frame->set_shortcut(ED_SHORTCUT("sprite_frames/delete", TTR("Delete Frame"), Key::KEY_DELETE));
+	copy->set_shortcut_context(tree);
+	copy->set_shortcut(ED_SHORTCUT("sprite_frames/copy", TTR("Copy Frame"), KeyModifierMask::CMD_OR_CTRL | Key::C));
+	paste->set_shortcut_context(tree);
+	paste->set_shortcut(ED_SHORTCUT("sprite_frames/paste", TTR("Paste Frame"), KeyModifierMask::CMD_OR_CTRL | Key::V));
+	empty_before->set_shortcut_context(tree);
+	empty_before->set_shortcut(ED_SHORTCUT("sprite_frames/empty_before", TTR("Insert Empty (Before Selected)"), KeyModifierMask::ALT | Key::LEFT));
+	empty_after->set_shortcut_context(tree);
+	empty_after->set_shortcut(ED_SHORTCUT("sprite_frames/empty_after", TTR("Insert Empty (After Selected)"), KeyModifierMask::ALT | Key::RIGHT));
+	move_up->set_shortcut_context(tree);
+	move_up->set_shortcut(ED_SHORTCUT("sprite_frames/move_left", TTR("Move Frame Left"), KeyModifierMask::CMD_OR_CTRL | Key::LEFT));
+	move_down->set_shortcut_context(tree);
+	move_down->set_shortcut(ED_SHORTCUT("sprite_frames/move_right", TTR("Move Frame Right"), KeyModifierMask::CMD_OR_CTRL | Key::RIGHT));
+
+	zoom_out->set_shortcut_context(tree);
+	zoom_out->set_shortcut(ED_SHORTCUT("sprite_frames/zoom_out", TTR("Zoom Out"), KeyModifierMask::CMD_OR_CTRL | Key::MINUS));
```

Comment:
Shouldn't the shortcut include `KP_MINUS` too?

Reply:
Yes, why not